themes/survey/fruity_twentythree/css/variations/theme_blueberry-rtl.css

```
@@ -11125,28 +11131,28 @@ body {
-  color: rgb(27.85, 175.5345714286, 194.95);
+  color: #1cb0c3;
 }
 .fruity_twentythree .footer {
   background-color: #ffffff;
   border-top: solid 1px #dadada;
 }
 .fruity_twentythree .alert-security-update {
   background-color: #ffffff;
-  border: 1px solid hsl(64.9714285714, 154%, 54.6862745098%);
-  color: hsl(64.9714285714, 154%, 54.6862745098%);
+  border: 1px solid #ecff18;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)